source/api/plugins/before-run-api.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Before Run API
```

Comment:
you know what? I'm just realizing all of these pages should be experimental, so we have a class that makes the page look 'experimental' - it basically makes the whole page orange. 

Can you add this to the other event pages also?

```suggestion
title: Before Run API
containerClass: experimental
```

Reply:
Going to do this in https://github.com/cypress-io/cypress-documentation/pull/3428